iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchingAffOrgQualityTest.java

```diff
@@ -253,6 +255,29 @@ private AffMatchingService createAffMatchingService() {
         
         affMatchingService.setAffOrgMatchers(of(docOrgRelationMatcher, mainSectionHashBucketMatcher, firstWordsHashBucketMatcher));
         
+        
+        AffMatchOrganizationAltNameFiller altNameFiller = createAffMatchOrganizationAltNameFiller();
+        affMatchingService.setAffMatchOrganizationAltNameFiller(altNameFiller);
+        
         return affMatchingService;
     }
+   
+   
+   private AffMatchOrganizationAltNameFiller createAffMatchOrganizationAltNameFiller() throws IOException {
+       
+       AffMatchOrganizationAltNameFiller altNameFiller = new AffMatchOrganizationAltNameFiller();
+       
+       List<String> altNamesCountryCodes = of(
```

Comment:
Again, if we had it in resource file we wouldn't need to define set of languages explicitly. I guess we could even be more generic and do not assume those are languages and simply put file names in registry. 

Going even further we could drop `universities_` prefix from file name since we have it all in `universities` directory.